builder/dockerfile/dispatchers.go

```diff
@@ -414,6 +414,8 @@ func cmd(b *Builder, args []string, attributes map[string]bool, original string)
 	}
 
 	b.runConfig.Cmd = strslice.StrSlice(cmdSlice)
+	// set config as already being escaped, this prevents double escaping on windows
+	b.runConfig.ArgsEscaped = true
```

Comment:
If you don't need it for Entrypoint then I'm really confused because its has args just like Cmd does.